### DISCUSSION TITLE
[2021.10.22] 네트워크

### DISCUSSION THREAD

**Discussion by pyunji** [pyunji/Hyundai-ITnE-Team_QuadCore#155] [Algorithm]
[네트워크](https://programmers.co.kr/learn/courses/30/lessons/43162)

**Comment by hyk339** [comment]
[네트워크 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/programmers/PR_43162.java)

**Comment by pyunji** [comment]
네트워크 unsolved
- [정리한 파일](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/study_files/DFS_BFS.md)

**Reply by leechang4** [reply]
고마워!!

**Reply by pyunji** [reply]
[네트워크 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/PR/PR_43162.java)
- [다른사람 풀이 참고함](https://velog.io/@ajufresh/%ED%94%84%EB%A1%9C%EA%B7%B8%EB%9E%98%EB%A8%B8%EC%8A%A4-%EB%84%A4%ED%8A%B8%EC%9B%8C%ED%81%AC-%EB%AC%B8%EC%A0%9C%ED%92%80%EC%9D%B4-Java)

**Comment by chaeyeon4u** [comment]
[네트워크 다른 풀이 참조](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/PR_43162.java)
[네트워크](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/PR_43162(2).txt)

**Reply by chaeyeon4u** [reply]
[네트워크](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/PR_43162(2).txt)
